From 880bad2c6b08afd2e2e303bc3ceea559edbe76d2 Mon Sep 17 00:00:00 2001
From: Kai Kang <kai.kang@windriver.com>
Date: Wed, 3 Oct 2018 00:27:11 +0800
Subject: [PATCH] cups: fix multilib install file conflicts

@CUPS_SERVERBIN@ is ${libdir} related that causes multilib install file
conflict. Remove @CUPS_SERVERBIN@ from the comment line of cups-files.conf to
avoid the conflict.

Upstream-Status: Inappropriate [OE specific]

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 conf/cups-files.conf.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/conf/cups-files.conf.in b/conf/cups-files.conf.in
index f96f745..27d8be9 100644
--- a/conf/cups-files.conf.in
+++ b/conf/cups-files.conf.in
@@ -70,7 +70,7 @@ PageLog @CUPS_LOGDIR@/page_log
 #RequestRoot @CUPS_REQUESTS@
 
 # Location of helper programs...
-#ServerBin @CUPS_SERVERBIN@
+#ServerBin
 
 # SSL/TLS keychain for the scheduler...
 #ServerKeychain @CUPS_SERVERKEYCHAIN@
